api/tests/unit/application/stages/index_test.js

```diff
@@ -36,4 +36,74 @@ describe('Unit | Router | stages-router', () => {
       expect(response.statusCode).to.equal(200);
     });
   });
+
+  describe('GET /api/admin/stages/:id', () => {
+    it('should exist', async () => {
+      // given
+      const method = 'GET';
+      const url = '/api/admin/stages/34';
+
+      // when
+      const response = await httpTestServer.request(method, url);
+
+      // then
+      expect(response.statusCode).to.equal(204);
+    });
+
+    it('should return a 400 error when payload does not exist', async () => {
+      // given
+      const method = 'PATCH';
+      const payload = { data: {
+        attributes: {
+          prescriberTitle: '',
+          prescriberDescription: undefined,
+        },
+      } };
+      const url = '/api/admin/stages/34';
+
+      // when
+      const response = await httpTestServer.request(method, url, payload);
+
+      // then
+      expect(response.statusCode).to.equal(400);
+    });
```

Comment:
**suggestion** : ce it PATCH est dans le contexte du GET, je pense qu'on peut le transformer en `should return a 400 error when id is not a number` ou un truc du genre